apps/codeforafrica/src/components/SocialMediaBar/SocialMediaBar.js

```diff
@@ -1,8 +1,8 @@
 import Stack from "@mui/material/Stack";
 import React from "react";
 
-const ShareBar = React.forwardRef(function ShareBar(props, ref) {
+const SocialMediaBar = React.forwardRef(function SocialMediaBar(props, ref) {
```

Comment:
```suggestion
const SocialMediaButtonGroup = React.forwardRef(function SocialMediaButtonGroup(props, ref) {
```

This just groups various `SocialMediaButton`s